src/app/child-dev-project/attendance/model/calculate-average-event-attendance.ts

```diff
@@ -32,8 +32,7 @@ export function calculateAverageAttendance(
   return {
     average:
       stats.get(AttendanceLogicalStatus.PRESENT) /
-      (stats.get(AttendanceLogicalStatus.PRESENT) +
-        stats.get(AttendanceLogicalStatus.ABSENT)),
+      event.children.length,
```

Comment:
I changed the computation of the average attendance. Otherwise the information in the `AttendanceCalendarComponent` is rather misleading (e.g. _100% Attended (of 24 Participants)_ even when some of the participants are skipped or not existing  any more (see #792)).

Reply:
adapted the participant count displayed there